src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, { ChangeEvent, FC, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+// eslint-disable-next-line max-len
+const DEFAULT_PREVIEW = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+type Props = {
+  movies: Movie[],
+  onAddMovie: (movies: Movie[]) => void,
+};
+
+export const FindMovie: FC<Props> = ({ movies, onAddMovie }) => {
+  const [query, setQuery] = useState<string>('');
```

Comment:
TS already knows about this type, bc you put an initial value
```suggestion
  const [query, setQuery] = useState('');
```